src/pages/signIn/signIn.stories.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import { SignIn } from './SignIn';
+
+
+export default { title: 'SignIn Page' };
+
+export const Default = () => <SignIn />;
```

Comment:
It might be good to add a note about how this components depends on environmental variables containing domain information. That way, knowledge of how to use this component is independent from prior knowledge about the application. Look at my Task example on how to add notes

Reply:
I will take a look when I have time

Reply:
I think is no need to explain that it depend on environment variable, because it must been set up in everyone's local repo, and I have share the environment variable in the slack